bootstrap/ansible/main.copyme

```diff
@@ -274,16 +278,13 @@ sentry_dsn: ""
 
 # # Email settings.
 # email_port: 1025
-# # TODO: Set these addresses to yours.
-# from_email: you@email.com
-# admin_email: you@email.com
+# from_email: placeholder@dev.dev
+# admin_email: placeholder@dev.dev
 # # TODO: For LRC, use the substring 'MyLRC'.
 # email_subject_prefix: '[MyBRC-User-Portal]'
 
 # # A list of admin email addresses to be notified about new requests and other
 # # events.
-# # TODO: Set these addresses to yours.
-# email_admin_list: ['you@email.com']
+# email_admin_list: []
```

Comment:
For this and the below, why not ['placeholder@dev.dev']?

Reply:
on staging and prod, I would guess that you add your own email addresses, whereas on dev a placeholder email can be added just to make sure that nothing breaks?